### DISCUSSION TITLE
Why this build failed when we update the forked repository from base repository?

### DISCUSSION THREAD

**Discussion by mhfaisalbd** [OpenCodeFoundation/eSchool#408] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/27910238/116778544-54d41900-aa94-11eb-9bc2-d8551bd251db.png)

**Comment by ratanparai** [comment, marked as answer]
The build is failing because your forked repo does not have the necessary secret setup for the DockerHub. 

Sorry for being late to answer your question.
